redaxo/src/core/lib/console/application.php

```diff
@@ -90,4 +94,23 @@ private function loadPackages(rex_console_command $command)
 
         rex_extension::registerPoint(new rex_extension_point('PACKAGES_INCLUDED'));
     }
+
+    private function checkConsoleUser(InputInterface $input, OutputInterface $output): bool
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        if (function_exists('posix_getuid')) {
+            $currentuser = posix_getpwuid(posix_getuid());
+            $webuser = posix_getpwuid(fileowner(rex_path::backend()));
```

Comment:
macht es sinn dass wir separat ne warning haben wenn es als root (z.b. mittels sudo) ausgeführt wird?

sudo ist ja in der regel "besonders falsch"?

Reply:
puhhh, weiß ich net, könnte man machen, ist mir persönlich net wichtig. Wenn es gewünscht ist, kann ichs gerne ergänzen.

Reply:
@gharlan meinung?